### PR TITLE
Correct history for `issued_to` + fix changelog for Estimates

### PR DESCRIPTION
**Pull request by pcothenet**
### What

- Correct the changelog for `issued_to`
- Correct the changelog for the names of the estimate methods
- Update the names of some internals

### Why

- Correctly document the breaking changes
- Fix the documentation

### SDK Release Checklist

- [ ] Have you added an integration test for the changes?
- [ ] Have you built the package locally and made queries against it successfully?
- [ ] Did you update the changelog?
- [ ] Did you bump the package version?
- [ ] If endpoints were removed, did you manually remove the corresponding files? (this should be rare)
- [ ] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,16 +9,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Breaking
+
+- Renames the `issuedTo` parameter for `orders` to `issued_to`.
 
 ## [1.23.0] - 2022-06-03
 
 ### Added
 
-- Adds support for the `issued_to` parameter on `orders`, to add support for creating and placing orders on behalf of another party.
+- Adds support for the `issuedTo` parameter on `orders`, to add support for creating and placing orders on behalf of another party.
```
Comment (pcothenet) [inline CHANGELOG.md:25]: @holtbp fixing the timeline here.

File: src/api/OrdersApi.js [hunk 2]
```diff
@@ -104,14 +104,10 @@ export default class OrdersApi {
...
-    const _placeOrderRequest = PlaceOrderRequest.constructFromObject(
-      opts,
-      new CreateOrderRequest()
-    );
-    let postBody = _placeOrderRequest;
+    let postBody = opts['placeOrderRequest'];
 
     // verify the required parameter 'id' is set
-    if (id === undefined || id === null) {
+    if (_id === undefined || _id === null) {
```
Comment (pcothenet) [inline src/api/OrdersApi.js:110]: I don't understand this change, this seems incorrect.

File: src/api/OrdersApi.js [hunk 3]
```diff
@@ -104,14 +104,10 @@ export default class OrdersApi {
   placeOrderWithHttpInfo(id, opts) {
     opts = opts || {};
 
-    const _placeOrderRequest = PlaceOrderRequest.constructFromObject(
-      opts,
-      new CreateOrderRequest()
-    );
-    let postBody = _placeOrderRequest;
+    let postBody = opts['placeOrderRequest'];
```
Comment (pcothenet) [inline src/api/OrdersApi.js:107]: Also strange, investigating.

File: src/model/CreateOrderRequest.js [hunk 4]
```diff
@@ -6,7 +6,7 @@
  */
 
 import ApiClient from '../ApiClient';
-import V1OrdersIssuedTo from './V1OrdersIssuedTo';
+import OrderIssuedTo from './OrderIssuedTo';
```
Comment (pcothenet) [inline src/model/CreateOrderRequest.js:9]: Avoids introducing this weird solo V1 file (this change is due to using a reusable schema for `order_issued_to` in swagger).

File: CHANGELOG.md (hunk 1 above)
Comment (pcothenet) [inline CHANGELOG.md:25]: @holtbp I'm rewriting the timeline here.

File: src/api/OrdersApi.js (hunk 3 above)
Comment (pcothenet) [reply src/api/OrdersApi.js:107]: Ah, that's the classic case of two 200 examples block overriding parts of the swagger file.

**Review by holtbp — APPROVED** [review]

**Pull request merged**